config/fedora.yml

```diff
@@ -0,0 +1,4 @@
+user: fedoraAdmin
```

Comment:
Guess it's to early to review this PR then since there's nothing about ActiveFedora here.